File: clients/java/examples/collections-get-collection-keyspace.java
import com.datastax.astra.client.DataAPIClient;
import com.datastax.astra.client.collections.Collection;
import com.datastax.astra.client.collections.CollectionOptions;
import com.datastax.astra.client.collections.definition.documents.Document;
import com.datastax.astra.client.databases.Database;

public class Example {

  public static void main(String[] args) {
    // Get a database
    Database database = new DataAPIClient("**APPLICATION_TOKEN**").getDatabase("**API_ENDPOINT**");

    // Get a collection
    CollectionOptions options = new CollectionOptions().keyspace("**KEYSPACE_NAME**");
    Collection<Document> collection = database.getCollection("**COLLECTION_NAME**", options);
  }
}

File: clients/go/examples/collections-get-collection-keyspace.go
package main

import (
	"github.com/datastax/astra-db-go/v2/astra"
	"github.com/datastax/astra-db-go/v2/astra/options"
)

func main() {
	// Get a database
	client := astra.NewClient()

	database := client.Database(
		"**API_ENDPOINT**",
		options.API().SetToken("**APPLICATION_TOKEN**"),
	)

	// Get a collection
	database.Collection(
		"**COLLECTION_NAME**",
		options.GetCollection().SetKeyspace("**KEYSPACE_NAME**"),
	)
}
